src/frame/TopAppBarFramePage.js

```diff
@@ -72,6 +72,18 @@ class TopAppBarFramePage extends Component {
     );
   }
 
+  getOffset(type = propToVariant.standard) {
+    let offsetClass = '';
+    if(type === propToVariant.standard || type === propToVariant.fixed) {
+      offsetClass = 'mdc-top-app-bar--fixed-adjust';
+    } else if (type === propToVariant['short-collapsed']) {
+      offsetClass = 'mdc-top-app-bar--short-fixed-adjust'
+    } else {
+      offsetClass = `${type.variant}-fixed-adjust`;
```

Comment:
Is this missing an extra dash (`${type.variant}--fixed-adjust`) ?

Reply:
No, this is correct for the variants.

Reply:
Dope! Thanks for clarifying.